packages/jitm/src/JITM.php

```diff
@@ -4,6 +4,7 @@
 
 use Automattic\Jetpack\Asset_Tools;
 use Automattic\Jetpack\Connection\Manager as Jetpack_Connection;
+use Automattic\Jetpack\Connection\Client;
```

Comment:
We can end up having more than one client (XMLRPC client, IXR client; should we start using this with an alias like `Connection_Client` or something?

Reply:
We can always disambiguate as needed, plus the methods that get called on this client illustrate pretty well what kind of client it is.

Reply:
Works for me!